conduct.md

```diff
@@ -53,11 +53,16 @@ Allegheny's honor code for all submitted work.
 
 * Acceptable Conduct (A):
   * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for deadlines.
+    * Turn assignments in on the due date, by the time set by JJ; the only exception to this rule is if the student is excused by the professor.
+    * Keep track of the due dates of the assignments.
+    * Be able to give reasons for missing deadlines.
```

Comment:
Suggestion: It might be worth including acceptable conduct for group work deadlines

Reply:
We added a bullet point about group work, how and when to turn it in.